katas/metal-slug/samuel/Soldier.js

```diff
@@ -0,0 +1,24 @@
+import { Civilian } from './Civilian.js';
+import { Enemy } from './Enemy.js';
+
+export class Soldier {
+    constructor() {
+        this.resistance = 1;
+        this.attackPower = 0;
+        this.lifes = 3;
+    }
+    changeWeapon(weaponAttackPower) {
```

Comment:
For next iteration, consider to receive an object here instead of an integer, so `changeWeapon` follows open/close principle